apps/verification-portal/src/lib/components/carousel/NewsCarousel.svelte

```diff
@@ -29,10 +29,14 @@
   function updateSlide(slide: number) {
     index = slide;
   }
-
+  let hasBeenClicked = false; // Prevents double click
   function toggleExpanded() {
+    if (hasBeenClicked) return;
+    hasBeenClicked = true;
     expanded = !expanded;
-
+    setTimeout(() => {
+      hasBeenClicked = false;
+    }, 200);
```

Comment:
For the future, I think we can just throttle through https://www.geeksforgeeks.org/lodash-_-throttle-method/